src/main/java/net/liquidpineapple/pang/logger/Logger.java

```diff
@@ -0,0 +1,77 @@
+package net.liquidpineapple.pang.logger;
+
+/**
+ * The logger class containing static methods to log out to the console.
+ * @author Jurriaan Den Toonder
+ * @date 2016/09/21.
+ */
+public class Logger {
+
+    private static int level;
```

Comment:
I would rename this variable to "option" or "optionPriority" to avoid confusion with pang/screens/Level.java